src/csharp/unitypackage/unitypackage_skeleton/Plugins/Grpc.Core/runtimes/grpc_csharp_ext_dummy_stubs.c

```diff
@@ -434,3 +434,16 @@ void grpcsharp_test_call_start_unary_echo() {
   fprintf(stderr, "Should never reach here");
   abort();
 }
+void dlopen() {
```

Comment:
I seems that the dlopen dlerror and dlsym will clash with the system-provided symbols when the "shared" scenarios is used? In another word, it seems that this might be fixing iOS, but it breaks Android in turn?

Reply:
I actually added these to fix the IL2CPP issues on Windows, as linked in this issue: https://github.com/grpc/grpc/issues/22485, where:
```
456A8C06ECBC58CA07B831FAD5D714FD.obj : error LNK2019: unresolved external symbol dlopen referenced in function Mono_dlopen_m90FC1077591F8256333A1484FF03CE4625E08CFE
456A8C06ECBC58CA07B831FAD5D714FD.obj : error LNK2019: unresolved external symbol dlerror referenced in function Mono_dlerror_m9C48C6C60464CAE063282826C532F97D421AC22A
456A8C06ECBC58CA07B831FAD5D714FD.obj : error LNK2019: unresolved external symbol dlsym referenced in function Mono_dlsym_mF4EC4FD7831724332FACF7ED28A13229C7B82946
E:\Projects\GRPC-IL2CPP\Library\il2cpp_cache\linkresult_F0C47798A0AFDCBB5F69B901A116EEF0\GameAssembly.dll : fatal error LNK1120: 3 unresolved externals
```

Reply:
I'm not too knowledgeable about differences between platforms - I do understand that these are dummies, but for which platforms would these three functions be missing for?